python/plugins/processing/algs/qgis/ui/InterpolationWidgets.py

```diff
@@ -327,8 +335,9 @@ def rowsChanged(self):
 
     def columnsChanged(self):
         cols = self.mColumnsSpinBox.value()
-        if cols < 2:
+        if cols < 2 or self.extent.isNull():
             return
+
         cell_size = self.extent.width() / (cols - 1)
```

Comment:
Same comment as above (check .width() != 0 ?) -- is there a good place to add a method to avoid the duplication ?

Reply:
All this code needs porting to C++ -- I'm only going to do the minimum here in order to fix the regression.